src/azure-cli-core/azure/cli/core/_output.py

```diff
@@ -33,7 +33,10 @@ class ComplexEncoder(json.JSONEncoder):
     def default(self, obj):  # pylint: disable=method-hidden
         if isinstance(obj, bytes) and not isinstance(obj, str):
             return obj.decode()
-        return json.JSONEncoder.default(self, obj)
+        try:
+            return json.JSONEncoder.default(self, obj)
+        except TypeError:
```

Comment:
I discussed with Travis, we will remove the try..catch and figured out what is broken w/o it.